### DISCUSSION TITLE
Unable to install any package

### DISCUSSION THREAD

**Discussion by b-mael** [Homebrew/discussions#2769] [Getting started]
Hello, 

I am on macOs Monterey with Intel. I install homebrew and I am on version :

Homebrew 3.3.10-21-g3ba6afb
Homebrew/homebrew-core (git revision 3ba6afb1d3b; last commit 2022-01-12)
Homebrew/homebrew-cask (git revision bccfa8bf06; last commit 2022-01-12)

When I try to install any package, for example brew install wget  or brew install rbenv, I have the error : 
No available formula with the name "wget"
No available formula with the name "rbenv"

For my work I have to install shopify-cli and it impossible ...

Someone can help me ?

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?

**Reply by b-mael** [reply]
brew config return : 

> HOMEBREW_VERSION: 3.3.10-21-g3ba6afb
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3ba6afb1d3b3272bc17bf8ff978e071cd35502b1
Last commit: 4 hours ago
Core tap ORIGIN: https://github.com/Homebrew/brew
Core tap HEAD: 3ba6afb1d3b3272bc17bf8ff978e071cd35502b1
Core tap last commit: 4 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit skylake
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.1-x86_64
CLT: 13.2.0.0.1.1637719136
Xcode: N/A

brew doctor return nothing except two warning at the beginning

**Reply by carlocab** [reply]
Please paste the warnings from `brew doctor` here.

**Reply by b-mael** [reply]
> Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

> Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

**Reply by carlocab** [reply]
That's very much the opposite of nothing. When asking for help online, please don't provide incomplete versions of the information that's requested from you. You're giving people who are trying to help you a much harder time when the information you've given them is incomplete.

Have you tried following this suggestion from `brew doctor`?

```
rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
brew tap homebrew/core
```

**Reply by b-mael** [reply]
There is this message above the warning, so no I don't : 

> Please note that these warnings are just used to help the Homebrew maintainers with debugging if you file an issue

I will try

**Reply by b-mael** [reply]
It's working perfectly, thank you very much ! 

Sorry for the inconvenience for brew doctor warning

**Reply by carlocab** [reply]
Sure; glad that's working for you.
